packages/components/molecules/f-media-element/src/config.js

```diff
@@ -0,0 +1,13 @@
+export const ALIGN = {
```

Comment:
Would rename this file `constants.js` to match the naming in other components.

Reply:
will rename